package.json

```diff
@@ -20,7 +20,7 @@
     "@radix-ui/react-slot": "^1.1.0",
     "@radix-ui/react-switch": "^1.1.1",
     "@react-oauth/google": "^0.12.1",
-    "@zk-email/sdk": "0.0.84",
+    "@zk-email/sdk": "0.0.85-10",
```

Comment:
Why are we moving to `85-10` versioning scheme? Also, what does it mean? Is the 10 subversion kind of thing?